src/Integration/WPF/ContextualCommandViewModel.cs

```diff
@@ -31,38 +36,50 @@ public ContextualCommandViewModel(object fixedContext, ICommand command)
                 throw new ArgumentNullException(nameof(fixedContext));
             }
 
-            if (command == null)
```

Comment:
The view model is shared but the commands are section specific, so we need to be able to change them to the active section.